OCPBUGS-19883: Move reboot warning in "Replacing an unhealthy etcd member" (#70035)
openshift/openshift-docs: modules/restore-replace-stopped-etcd-member.adoc

----
+
This command ensures that you can successfully re-create secrets and roll out the static pods.
+
[IMPORTANT]
====
After you turn off the quorum guard, the cluster might be unreachable for a short time while the remaining etcd instances reboot to reflect the configuration change.
====
+
[NOTE]
====

> etcd cannot tolerate any additional member failure when running with two members. Restarting either of the two remaining members breaks quorum and causes downtime in your cluster. The quorum guard protects etcd from restarts due to configuration changes that could cause such downtime, which is why it must be disabled to complete this procedure.

Reviewer: ```suggestion
etcd cannot tolerate any additional member failure when running with two members. Restarting either remaining member breaks the quorum and causes downtime in your cluster. The quorum guard protects etcd from restarts due to configuration changes that could cause downtime, so it must be disabled to complete this procedure.
```
Suggestion: Minimalism - See SSG https://redhat-documentation.github.io/supplementary-style-guide/#minimalism
